nautobot/core/celery/task.py

```diff
@@ -1,12 +1,85 @@
-from celery import Task
-
-from nautobot.extras.jobs import get_task_logger
-
-logger = get_task_logger(__name__)
+from billiard.einfo import ExceptionInfo, ExceptionWithTraceback
+from celery import states, Task
+from celery.exceptions import Retry
+from celery.result import EagerResult
+from celery.utils.functional import maybe_list
+from celery.utils.nodenames import gethostname
+from kombu.utils.uuid import uuid
 
 
 class NautobotTask(Task):
     """Nautobot extensions to tasks for integrating with Job machinery."""
 
+    def apply(
```

Comment:
This is just moved from `BaseJob.apply` to `NautobotTask.apply` since BaseJob isn't a task any more.